snippets/snippets.json

```diff
@@ -52,5 +52,50 @@
             "}"
         ],
         "description": "A hasOneThrough relationship function"
+    },
+    "A MorphOne relationship function": {
+        "prefix": "morphOne",
+        "body": [
+            "public function ${1:name}()\n{",
+            "    return \\$this->morphOne(${2:class}, '${1:name}able');",
+            "}"
+        ],
+        "description": "A morphOne relationship function"
+    },
+    "A MorphTo relationship function": {
+        "prefix": "morphTo",
+        "body": [
+            "public function ${1:nameable}()\n{",
+            "    return \\$this->morphTo(${2});",
+            "}"
+        ],
+        "description": "A morphTo relationship function"
+    },
+    "A MorphMany relationship function": {
+        "prefix": "morphMany",
+        "body": [
+            "public function ${1:names}()\n{",
+            "    return \\$this->morphMany(${2:class}, '${3:nameable}');",
+            "}"
+        ],
+        "description": "A morphMany relationship function"
+    },
+    "A MorphToMany relationship function": {
+        "prefix": "morphToMany",
+        "body": [
+            "public function ${1:names}()\n{",
+            "    return \\$this->morphToMany(${2:class}, '${3:nameable}');",
```

Comment:
same issue with "nameable"

Reply:
I kept it as a separate focus because of following:

```php
    public function tags()
    {
        return $this->morphToMany(Tag::class, 'taggable');
    }
```

Reply:
Yes, it will not work in all situations, but at least it will cover most of the use-cases.